web/packages/teleport/src/services/audit/types.ts

```diff
@@ -120,6 +120,7 @@ export const eventCodes = {
   DEVICE_ENROLL: 'TV005I',
   DEVICE_AUTHENTICATE: 'TV006I',
   DEVICE_UPDATE: 'TV007I',
+  DEVICE_WEB_TOKEN_CREATE: 'TV008I',
```

Comment:
We usually use a separate code for the failure case. Any reason not to do that here?

Reply:
All device events are like this. For this particular one we only record successes, so it won't make much of a difference.